config/locales/es.yml

```diff
@@ -449,6 +449,13 @@ es:
         header: "¡Declare sus impuestos por su cuenta de forma gratuita!"
         send_link: Envíenme un enlace
         subheader: Brindamos acceso gratuito a un sitio de preparación de impuestos en línea llamado TaxSlayer. Le enviaremos un enlace para crear su propio acceso a TaxSlayer, para que pueda presentarlo por su cuenta.
+      eip_home:
+        header: Reciba su cheque de estímulo
+        questions_cta: la página de preguntas frecuentes de estímulo
+        questions_html: Preguntas? Visite %{faq_link}
+        subheader1: Es posible que se esté perdiendo un pago de más de $ 1,200 dólares. Los voluntarios entrenados y certificados por el IRS están listos para ayudarle a presentar su declaración de forma gratuita.
```

Comment:
Pedantic, but from what I read elsewhere, should it be `$1,200 ` not `$ 1,200`? Here's a sample web page that calls it `$1,200` https://www.ssa.gov/espanol/coronavirus/

Feel free to merge as-is. Feel free to change this and merge without requesting further review.

Reply:
ah yes i guess that space got added in the translation somehow. i will update and merge. thanks!